### ISSUE TITLE
Pause slider upon manual slide change

### ISSUE THREAD

**Issue by maschad96** [wnolfm/a11y-slider#8]
James Jackson left a meaningful piece of feedback in the w3c/aria-practices Carousel project that I think should be considered in this project

[His suggestion](https://github.com/w3c/aria-practices/issues/970#issuecomment-515632131)
